### ISSUE TITLE
[Bug]: side-nav visual hierarchy

### ISSUE THREAD

**Issue by MatissJanis** [actualbudget/actual#2944]
### Verified issue does not already exist?

- [X] I have searched and found no existing issue
- [X] I will be providing steps how to reproduce the bug (in most cases this will also mean uploading a demo budget file)

### What happened?

<img width="372" alt="Screenshot 2024-06-26 at 17 31 14" src="https://github.com/actualbudget/actual/assets/886567/821d2d3b-9d52-425c-aa15-c788eff59672">

The addition of the underline for account categories (introduced here https://github.com/actualbudget/actual/pull/2847) has created an issue: **the visual hierarchy** between accounts and top-level items has been broken.

Some potential solutions:
- slightly increase the font size for "off budget/on budget/all accounts" links and remove the underline
- uppercase the "off budget/on budget/all accounts" links and remove the underline
- underline "all accounts" too (this might look ugly)
- ???

### Where are you hosting Actual?

None

### Operating System

None

**Comment by ParthJohri** [comment]
Hi @MatissJanis , Thank you for the bug clarification, I have made the following changes (screenshot attached), pleasre let me know if any further changes need to be done. If everything looks good, then I will create the PR for it.

### Modifications
1. Went on with your first suggestion of slightly Increaseing the font size for "off budget/on budget/all accounts" links and removed the underline. All Accounts has slightly more font size than off budget/on budget.
2. Capitalised each word's first letter to put emphasis on the title.

### Screenshots 
![navbar-heirarchy](https://github.com/actualbudget/actual/assets/76129377/db69a350-3ac5-4e75-8ef0-6c7e40d0ca03)

Thank you!

**Comment by youngcw** [comment]
I like the font size increase and the added capitalization.  I do still like the underline though if there is a way to keep it and not make the All Accounts header get lost

**Comment by MatissJanis** [comment]
Agreed with @youngcw on all things except the underline. IMO it stands out too much in the side-nav and thus creates unnecessary clutter. 

We'll need a tie-breaker here on this topic :)

**Comment by ParthJohri** [comment]
@MatissJanis @youngcw 
How about this one?
1. Using a solid border to show a parent hierarchy of all accounts.
2. Using a dotted border to show for budget/ off budget.

![navbar-heirarchy2](https://github.com/actualbudget/actual/assets/76129377/cf508da8-3f33-40ac-9555-818472252c67)

**Comment by newtonfav** [comment]
The dotted border doesn't look nice. Why not use a solid border with a very thin border-width? Additionally, remove the solid border for the All Accounts header and increase its font size.

**Comment by ParthJohri** [comment]
> The dotted border doesn't look nice. Why not use a solid border with a very thin border-width? Additionally, remove the solid border for the All Accounts header and increase its font size.

@newtonfav Kind of like this?
![navbar-heirarchy3](https://github.com/actualbudget/actual/assets/76129377/b751b710-f3dc-4f34-a37f-6288cacf98fc)

**Comment by youngcw** [comment]
thats pretty nice.

**Comment by ParthJohri** [comment]
> thats pretty nice.

If the above looks good, I can create a PR for it, please let me know if there is any other change to be made.

**Comment by MatissJanis** [comment]
"All accounts" is too large. None of these nav items should exceed in size the "Setting/Reports/etc" nav items. 

How would it look in lightmode? I'm still on the fence for the underlines.

**Comment by ParthJohri** [comment]
> "All accounts" is too large. None of these nav items should exceed in size the "Setting/Reports/etc" nav items.
> 
> How would it look in lightmode? I'm still on the fence for the underlines.

@MatissJanis 
- If the font size can't be increased, how about using bullet points?
For example:-

### All Accounts
 - #### For Account
 - #### Off Account 
 
### Screenshots for all the themes:- 
#### Light Mode
![Screenshot 2024-06-28 at 02 37 34](https://github.com/actualbudget/actual/assets/76129377/d723789c-3aa2-44e6-9d6b-ed15a11b5df4)
#### Dark Mode
![Screenshot 2024-06-28 at 02 37 50](https://github.com/actualbudget/actual/assets/76129377/eec14c83-61df-40fb-8e6a-b58dfaaad206)
#### Midnight Mode
![Screenshot 2024-06-28 at 02 38 01](https://github.com/actualbudget/actual/assets/76129377/f39b6e95-427f-4dc6-bd1d-3e5964a03f16)

**Comment by Teprifer** [comment]
Not sure about bullet points, the account names would also have to be indented a fair bit. 

One idea put forward in discord was to only slightly indent the account names by 5px.

**Comment by MatissJanis** [comment]
Of all the revisions: I liked best the 1st one and second-best was the last one.

Happy to go with either of those options if people still feel like the underline is a good idea for the UX.

**Comment by ParthJohri** [comment]
> Of all the revisions: I liked best the 1st one and second-best was the last one.
> 
> Happy to go with either of those options if people still feel like the underline is a good idea for the UX.

@MatissJanis @youngcw @Teprifer @newtonfav 
How does this look?
1. Indentation of 5 pixels for accounts, sub-accounts.
2. Same font size but capitalisation of each account word.
3. Horizontal line for the accounts, excluding the All Accounts.

![navbar-ui-change](https://github.com/actualbudget/actual/assets/76129377/239b5d70-31d4-4c09-adac-8a500012155b)

**Comment by MatissJanis** [comment]
Comparatively I prefered the 1st version best. The horizontal indentation looks out of place.

**Comment by ParthJohri** [comment]
> Comparatively I prefered the 1st version best. The horizontal indentation looks out of place.

Thank you @MatissJanis for your suggestions, if others agree then I can create the PR for it.

**Comment by youngcw** [comment]
@ParthJohri Go ahead and make a PR we can keep discussing there if needed

**Comment by matt-fidd** [comment]
@MatissJanis do you still stand by this? absolutely fine if you do, it's just been a while :)

**Comment by MatissJanis** [comment]
Lets close this. We ca revisit when/if we redesign the sidenav.
